### PR TITLE
fix(chat): fix sticker button not showing + stickers not showing in chat

### LINKED ISSUE #4407: [base_bc] stickers UI missing
# Bug Report

## Description

Stickers UI is missing in the base_bc branch, there is the gif selector and emojis but the stickers button is missing

### Additional Information

base_bc branch

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #4407

### CODE REVIEW

File: ui/app/AppLayouts/Profile/stores/AdvancedStore.qml
Comment (saledjenic) [inline ui/app/AppLayouts/Profile/stores/AdvancedStore.qml:35]: @jrainville personally I don't like this mess here in `AdvancedStore`, why don't we be uniform and do it in the same way we did for `mainModule`:

- add this to `Global`:
```
property var advancedModuleInst
property string currentNetworkId: ""
onAdvancedModuleInstChanged: {
    currentNetworkId = advancedModuleInst.currentNetworkId;
}
```
- there is `onAppStateChanged` slot in `ui/main.qml` and there you just add `Global.advancedModuleInst = profileSectionModule.advancedModule`

That's all.

**Review by saledjenic — APPROVED** [review]
@jrainville I am approving this, if you have time wold be good to apply update from a comment below.

File: ui/imports/utils/Global.qml
Comment (alexandraB99) [inline ui/imports/utils/Global.qml:19]: This property is [part of Global](https://github.com/status-im/status-desktop/blob/a93ebe3366f1684336993bd091945ad224bec895/ui/imports/utils/Global.qml#L18), would mind using that one instead?

File: ui/app/AppLayouts/Profile/stores/AdvancedStore.qml
Comment (alexandraB99) [reply ui/app/AppLayouts/Profile/stores/AdvancedStore.qml:35]: `Global` is not a store hence it shouldn't keep any instance of the backend, same for `main.qml`, shouldn't directly use `profileSectionModule.advancedModule`. I don't see any problem of this happening here which aligns well with the structure we attempt to maintain in QML.

Comment (jrainville) [reply ui/app/AppLayouts/Profile/stores/AdvancedStore.qml:35]: So is it better to leave it as is or change it like Sale recommended. I personally don't care that much

**Comment by jrainville** [comment]
@saledjenic @alexandraB99 I just pushed a new commit that addresses Sale's comment, but I don't mind reverting if needed.
I also added code to fix the stickers in the chat. I had not realized that it wasn't hooked in the message_item. Now stickers show correctly (and not the placeholder)
Please give another review when you have the time. 
I'll squash once I have the approvals.

File: ui/app/AppLayouts/Profile/stores/AdvancedStore.qml
Comment (alexandraB99) [reply ui/app/AppLayouts/Profile/stores/AdvancedStore.qml:35]: It's as is, as it's not introducing any regressions

**Review by alexandraB99 — APPROVED** [review]

**Review by saledjenic — APPROVED** [review]

**Review by abhi-status — APPROVED** [review]
Fixed & works on both Win 11 & Linux.

**Pull request merged**
